source/lambda/documentbulkprocessor/lambda_function.py

```diff
@@ -86,9 +98,16 @@ def processDocument(record):
 
     # error trying to dowload document, exit
     if ret == False:
-        print("ProcessDocument: failed to download locally document:" + ingestionDocumentKey)
+        print("ProcessDocument: failed to download locally document:" +
+              ingestionDocumentKey)
         return
-    
+
+    # check is file type is of a valid type (png, jpg, or pdf)
+    if is_invalid_mime_type(ingestionDocumentFilename):
```

Comment:
FYI:
We have a check for file type in document processor since that covers both the bulkprocessor and UI uploaded document paths and provides a status update on the UI (document comes up as "NOT SUPPORTED") 
https://github.com/awslabs/document-understanding-solution/blob/7be3bfce8775909cdaffb4e87b0d78286b7874e4/source/lambda/documentprocessor/lambda_function.py#L52

Reply:
it seemed like a good idea to stop files as high in the funnel as possible, which is hard for the standard funnel as documents are uploaded to S3 separately from the API call. However, it does mean repeating the function code if we don't refactor the helper lambda.